src/index.html

```diff
@@ -11,7 +11,146 @@
     <link rel="stylesheet" href="./style.css">
   </head>
   <body>
-    <h1>HTML Form</h1>
+    <form action="https://mate-academy-form-lesson.herokuapp.com/create-application"
+    method="POST"
+    onsubmit="onSubmit()">
+      <fieldset>
+        <legend>
+          Personal information
+        </legend>
+        <div class="form-field">
+          <label for="surname">Surname:</label>
+          <input id="surname"
+          type="text"
+          name="surname"
+          autocomplete="off">
```

Comment:
Same here - checklist item 3
Need to fix it all over your document.
```suggestion
          <input
            id="surname"
            type="text"
            name="surname"
            autocomplete="off"
          >
```